### ISSUE TITLE
Is it possible to recognize submissions before installing the extension?

### ISSUE THREAD

**Issue by imxiaobo** [HaolinZhong/PMCA#4]
Is it possible to recognize submissions before installing the extension? The review list contains LC problems only after installing the extension.

**Comment by HaolinZhong** [comment]
Yes, it is possible by utilizing the LeetCode submission API. I've added this feature to the roadmap.

**Comment by kagetora0924** [comment]
Is there any speculation about when this feature may become available?

**Comment by HaolinZhong** [comment]
> Is there any speculation about when this feature may become available?

After refactoring PMCA, I will work on this feature. The estimated release time for this feature would be March 2024. (hope I got time to finish it before this date...
